[2022-04-29 15:33:35,446] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:33:34.007459+00:00 [queued]>
[2022-04-29 15:33:35,469] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:33:34.007459+00:00 [queued]>
[2022-04-29 15:33:35,471] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:33:35,472] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-04-29 15:33:35,473] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 15:33:35,492] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): import_cat.check_presence_of_cat2> on 2022-04-29 15:33:34.007459+00:00
[2022-04-29 15:33:35,501] {standard_task_runner.py:52} INFO - Started process 3860 to run task
[2022-04-29 15:33:35,509] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cdn_daily', 'import_cat.check_presence_of_cat2', 'manual__2022-04-29T15:33:34.007459+00:00', '--job-id', '1715', '--raw', '--subdir', 'DAGS_FOLDER/cdn_daily.py', '--cfg-path', '/tmp/tmp847d447r', '--error-file', '/tmp/tmpi6bsbqtp']
[2022-04-29 15:33:35,512] {standard_task_runner.py:77} INFO - Job 1715: Subtask import_cat.check_presence_of_cat2
[2022-04-29 15:33:35,646] {logging_mixin.py:109} INFO - Running <TaskInstance: cdn_daily.import_cat.check_presence_of_cat2 manual__2022-04-29T15:33:34.007459+00:00 [running]> on host d5ad096ffea5
[2022-04-29 15:33:35,791] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cdn_daily
AIRFLOW_CTX_TASK_ID=import_cat.check_presence_of_cat2
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T15:33:34.007459+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-29T15:33:34.007459+00:00
[2022-04-29 15:33:35,836] {python.py:175} INFO - Done. Returned value was: None
[2022-04-29 15:33:35,866] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=cdn_daily, task_id=import_cat.check_presence_of_cat2, execution_date=20220429T153334, start_date=20220429T153335, end_date=20220429T153335
[2022-04-29 15:33:35,928] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 15:33:36,020] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
